Makefile

```diff
@@ -106,6 +106,13 @@ ifeq ($(E2E_SKIP_BUILD), 0)
 	docker push $(LOCAL_REGISTRY)/jmalloc/echo-server:latest
 endif
 
+# rebuild ingress controller image and push to the local registry for e2e tests.
```

Comment:
Why reduplicate the same contents that were implemented in another make directive (`push-images-to-kind`)?

Reply:
Because when I'm debugging the e2e, I only modify the ingress controller code. So I want to only rebuild and push this image to re-run e2e tests for faster. I will remove this.